CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Adds JSON Serialization library for Python. #1186
 - Adds PHP League Authentication Provider for PHP #1201
 - Added Shell language support #738
+- Added support for global error handling in python abstractions #1289
```

Comment:
move this to the added section that's under the unreleased tag please

Reply:
Oops. My bad.